packages/mock-account-provider/app/lib/transactions.server.ts

```diff
@@ -0,0 +1,72 @@
+import { mockAccounts } from './accounts.server'
+import { getPaymentPointerPayments } from './requesters'
+
+export interface Amount {
+  value: bigint
+  assetCode: string
+  assetScale: number
+}
```

Comment:
Since I also added that in [my last MAP PR](https://github.com/interledger/rafiki/blob/24f486145000fb908d712ebe43bca10cab01313a/packages/mock-account-provider/app/routes/webhooks.ts#L22-L26), should we share that?